cli/interface.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2018 Huawei Corporation.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli"
+	"text/tabwriter"
+
+	vc "github.com/kata-containers/runtime/virtcontainers"
+)
+
+var interfaceCLICommand = cli.Command{
+	Name:  "interface",
+	Usage: "manage interfaces for container",
+	Subcommands: []cli.Command{
+		infAddCommand,
+		infListCommand,
+		infUpdateCommand,
+	},
+	Action: func(context *cli.Context) error {
+		return cli.ShowSubcommandHelp(context)
+	},
+}
+
+var infAddCommand = cli.Command{
+	Name:      "add",
+	Usage:     "add a new nic into a container",
+	ArgsUsage: "<container-id> <interface-name>",
```

Comment:
We need to discuss how we see this working. Do we expect the caller to provide the whole interface (that's the declarative model we've been talking about), or do we simply provide an interface name and we expect virtcontainers to scan the network namespace to get all the information about this interface ?